.travis.yml

```diff
@@ -148,11 +148,6 @@ jobs:
       env: >-
         FILE_ENV="./ci/test/00_setup_env_native_valgrind.sh"
 
-    - stage: test
-      name: 'x86_64 Linux  [GOAL: install]  [bionic]  [no depends, only system libs, c++17]'
```

Comment:
Maybe add a c++17 marker to the qt5 job it's merged with?

Reply:
Thanks, done